react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,19 +1,26 @@
 import React, { Component } from 'react';
+import { array, func } from 'prop-types';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
 class Board extends Component {
-  renderSquare() {
-    return <Square />;
+
+  renderSquare(i) {
+    return (
+      <Square
+        value={this.props.squares[i]}
+        onClick={this.props.onClick}
```

Comment:
For no repeate this.props you can extract the keys.

const { squares, onClick } = this.props;